FreeRTOS/FreeRTOS-Plus-TCP
# Add words that are marked incorrectly spelled by spell checker to dictionary

Description
-----------


Test Steps
-----------
Add words that are marked incorrectly spelled by spell checker to dictionary

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
